src/new_reliquary/core.clj

```diff
@@ -27,5 +30,15 @@
   ([category transaction-name callback]
     (with-newrelic-transaction category transaction-name {} callback)))
 
+(defn with-successfully-newrelic-transaction
```

Comment:
Make ignore-transaction the default behaviour for with-newrelic-transaction instead of adding a new one